test/E2ETests/remoteDeploymentConfig.json

```diff
@@ -0,0 +1,9 @@
+﻿{
+  "NanoServer": {
```

Comment:
I wanted it to be there for anyone to know about how the config keys should be created via environment variables.